apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -415,16 +415,17 @@ class JsDebugger extends React.Component {
   };
 
   render() {
-    const {appType, isAttached, canRunNext, isRunning} = this.props;
+    const {
+      appType,
+      isAttached,
+      canRunNext,
+      isRunning,
+      debugButtons
+    } = this.props;
     const hasFocus = this.props.isDebuggerPaused && !this.props.isEditWhileRun;
 
     const canShowDebugSprites = appType === 'gamelab';
 
-    const sliderStyle = {
-      marginLeft: this.props.debugButtons ? 5 : 45,
-      marginRight: 5
-    };
-
     const openStyle = {};
     if (!this.state.open && this.state.transitionType !== 'closing') {
```

Comment:
I couldn't figure out a way to move `openStyle` into css given that `height` is passed in as a prop. If there's a way to do this let me know!